### DISCUSSION TITLE
Pmapper graph output fetch : using Cloudfox with STS role

### DISCUSSION THREAD

**Discussion by hackncode** [BishopFox/cloudfox#27] [General]
Hi @sethsec-bf 

I was using a STS ec2 role with permissions defined for cloudfox to enumerate one of my target cloud accounts. So I don't use any flag for profile while running any tools. 

I ran pmapper successfully without any issues.

`pmapper graph create`

I was able to visualize the graph output.

After this I tried to run all checks and individual checks with cloudfox

When I checked output, I found cloudfox is unable to find the pmapper graph output.
Which I have generated previously.

```
└─# ./cloudfox aws -v2 pmapper
[🦊 cloudfox v1.10.0-prerelease 🦊 ] AWS Caller Identity: arn:aws:sts::987654321012:assumed-role/Test-Role/i-12345678901c23kv9
[pmapper][987654321012-ABCDEFGGH7JY5XKPTROPN_i-12345678901c23kv9] Looking for pmapper data for this account and building a PrivEsc graph in golang if it exists.
[pmapper][987654321012-ABCDEFGGH7JY5XKPTROPN_i-12345678901c23kv9] No pmapper data found for this account.
1. Generate pmapper data by running `pmapper --profile 987654321012-ABCDEFGGH7JY5XKPTROPN_i-12345678901c23kv9 graph create`
2. After that completes, this cloudfox command and others will attempt to find and use the pmapper graph data in this and other cloudfox commands

```

How do I resolve this ?
Is there a way to manually input this pmapper graph data to cloudfox.

Thanks for the awesome tool 🔥

**Comment by sethsec-bf** [comment]
Thanks for sharing this @hackncode. This is actually a bug that affects all linux installs and not even related to the STS.  I'm going to fix this and get it in main this week and create a new release.  That should fix your issue but i still might want to give the user a way to supply the paths as arguments to cloudfox, so i might implement that at the same time. Hang tight, and thanks for finding the bug!

**Comment by sethsec-bf** [comment]
@hackncode Can you build from the main branch now and run it again to see if it fixes your issue?

**Reply by hackncode** [reply]
@sethsec-bf I ran the new build from main branch.
Looks like the issue is resolved now. Cloudfox is able to automatically ingest pmapper graph data and give out intended output.

Thank you very much for resolving the issue.
